tn_api_pricing_booking_spec.yml

```diff
@@ -141,7 +141,7 @@ paths:
                                         type: array
                                         example: ['Invalid agency/agent entered']
 
-    /book/trip/{trip_id}:
+    /book/trip/{trip_id}/?agency={agency}:
```

Comment:
```suggestion
    /book/trip/{trip_id}?agency={agency}:
```

Reply:
It actually is the way I wrote it:
I copied the list endpoint 
book/list/?agent ..... etc

Reply:
So you'd say the URL actually works like this? I doubt that will work 🤔